### ISSUE TITLE
On Try Again spell if local or remote connection is retried

### ISSUE THREAD

**Issue by dilyanpalauzov** [openhab/openhab-android#3417]
The screenshot says: The connection with the host was not successful. [Try again]

In openhab-Android I have a local and a remote connection.  When I am in the home network, I cannot connect to the remote connection.  This is not an openHab  problem, but a matter of routing fact.

When in this state, and I click on [Try again] openhab-Android tries to reconnect, but does not spell, whether it tries to reconnect fist to the local and then to the remote connection.  My colclusion is, that it tries to connect only to the remote connection.

* the programme shall try to reconnect to both local and remote connections
* it shall display to the user which connection it is currently trying to utilize.

![Screenshot_20230612_113519_org openhab habdroid beta](https://github.com/openhab/openhab-android/assets/4992947/de2f2968-953f-49af-a31e-16f186e73803)

**Comment by dilyanpalauzov** [comment]
> > it shall display to the user which connection it is currently trying to utilize.
> 
> At the time of pressing 'retry', it doesn't even know that yet (see above).

Once the programme knows, it shall show it, using a snackbar or however it was called.

**Comment by maniac103** [comment]
> Once the programme knows, it shall show it, using a snackbar or however it was called.

If you configured both local and remote connections, it already does.

**Comment by dilyanpalauzov** [comment]
Yes, it does show it on start.  But from the above screenshot state it did not show it.

I observed this several times and I will not be soon in that network to retry it again.

**Comment by dilyanpalauzov** [comment]
With openhab-Android 3.7.0 I changed the remote URI to unresolvable name and created random local URI (http://192.168.168.168).  I connect over public Wi-Fi.  The error message is “Cannot connect to the domain”.  (The error message in the screenshot above is: The connection with the host is unsuccessful.  So not exactly the same)

I click on Try again.  The system does not show a hint, whether it tries to connect first to the local and then to the remove address.
